### PR TITLE
Check if the hashes correspond if a hash is already present

### PR DESCRIPTION
**Pull request by AdminXVII**
- If the field already has text, verify that the hash matches it
- Add a Check button, rather than performing a check per dropdown
selection: avoids simultaneously running checks and let the user decide
when to start the check

TODO: Nicer theme?

Fix: #68

### CODE REVIEW

File: gtk/src/app/views/mod.rs [hunk 1]
```diff
@@ -26,6 +26,7 @@ pub struct Content {
 impl Content {
     pub fn new() -> Content {
         let image_view = ImageView::new();
+        image_view.set_hash_sensitive(false);
```
Comment (mmstick) [inline gtk/src/app/views/mod.rs]: If it isn't defaulting to false, perhaps we should make it the default when constructing the image view?

Comment (AdminXVII) [reply gtk/src/app/views/mod.rs]: I thought it was redundant, so I removed it, but sure, there is no real reason not to

Comment (AdminXVII) [reply gtk/src/app/views/mod.rs]: Done

**Comment by mmstick** [comment]
Screenshot from 2019-05-01 15-05-23
 It works

**Pull request merged**